src/wiki/templatetags/wiki_tags.py

```diff
@@ -52,8 +52,11 @@ def wiki_render(context, article, preview_content=None):
 
     if preview_content:
         content = article.render(preview_content=preview_content)
+    elif article.current_revision:
+        content = article.get_cached_content(context.get('user'))
```

Comment:
Since `user` is a kwarg, it's best to call it with `user=context.get('user')`

Reply:
fixed